src/calendar/day/multi-period/index.js

```diff
@@ -70,7 +70,13 @@ class Day extends Component {
             marginRight: 4
           });
         }
-        return <View key={index} style={style}/>;
+        return (
+           <View key={index} style={style}>
+              validPeriods.map((period, index) => {
+              <Text style={{fontWeight:'600', fontSize:12}}>{period.name}</Text>
```

Comment:
If you add a 'name' you should declare it, otherwise, no one will know that there is such an option.

Reply:
Also, inline style is bad for performance and you should provide a way to override it.